api/src/controllers/publications.js

```diff
@@ -12,7 +12,9 @@ const logger = require('winston');
 
 const { firefox } = require('playwright');
 
-const { playwrightConfig, categoryType } = require('../config/playwright');
+const { categoryTypes } = require('../config/publication');
+
+const { playwrightConfig } = require('../config/playwright');
 
 const { fillErrorObject } = require('../middleware/error');
```

Comment:
Could you group together the relative imports so there's no newline between them to maintain consistency? E.g. https://github.com/Researchify/Researchify/blob/main/api/src/controllers/publications.js#L9-L12